### DISCUSSION TITLE
Learn

### DISCUSSION THREAD

**Discussion by ClickOneRose** [community/community#28840] [GitHub Education]
Hi my name is Rose Pina, 
I think I push the wrong button at the beginning of my joining Git Hub. I want to learn automation, I'm taking a course at HubSpot, I was speaking to someone they told me they had some good programs here. I'm very new at this coding, I was just wondering if I made a mistake when I put student to join GitHub? I don't know if HubSpot qualifies me as a student. I don't want to break policies on something I don't understand if it qualifies me to be here at GitHub or not. I love this platform, but I want to choose a couple things besides coding, if they have that? I'm going in circles, because I don't know exactly this platform my first time being here. If someone can help me I would appreciate it. Thank you.
clickonerose@aol.com

**Comment by radar-blip** [comment]
Stay tuned Rose. Maybe Staff or another member may see this and have an answer for you. 
I myself am not a student so I dont know what qualifications you will need.

**Reply by ClickOneRose** [reply]
Okay Thank Mr. Lawson
